mods/stairs/init.lua

```diff
@@ -82,13 +84,15 @@ function stairs.register_stair(subname, recipeitem, groups, images, description,
 			end
 		end
 	end
+
 	local new_groups = table.copy(groups)
 	new_groups.stair = 1
 	warn_if_exists("stairs:stair_" .. subname)
 	minetest.register_node(":stairs:stair_" .. subname, {
 		description = description,
 		drawtype = "nodebox",
 		tiles = stair_images,
+		use_texture_alpha = src_def and src_def.use_texture_alpha,
```

Comment:
But this field will not be set to `true` in the nodedef of cubic nodes that contain transparent pixels. For example the field is not present in default:glass.
Maybe this needs to be a new stairs API field?

Reply:
Well, after this PR it is present in `default:glass` so that it can be inherited. I'd rather revert these changes than extending the stairs API right now. It seems very fragile and would need a sane implementation using definition tables rather than adding function arguments.

Reply:
> Well, after this PR it is present in default:glass so that it can be inherited

Uh sorry, i forgot that was part of this PR.
Nevermind this comment then, what you have done is simpler than adding an API field.
No problem.